src/components/sidebar-menu-item/sidebar-menu-item.js

```diff
@@ -11,8 +11,8 @@ export const SidebarMenuItem = ({ active, isTitle, children, onClick }) => {
 
   return (
     <li
-      className={`__sidebar-menu-item ${active && "active"} ${isTitle &&
-        "isTitle"}`}
+      className={`__sidebar-menu-item ${active ? "active" : ""} 
+      ${isTitle ? "isTitle" : ""}`}
```

Comment:
nit: We could use `classnames` as we already have it in the project.
```js
import classNames from "classnames";

className={classNames("__sidebar-menu-item", {
  "isTitle": isTitle,
  "active": active
})}
```

or even cleaner

```js
className={classNames("__sidebar-menu-item", {
  isTitle,
  active
})}
```

Reply:
Salut! Am modificat